src/DropboxAdapter.php

```diff
@@ -60,6 +62,11 @@ public function fileExists(string $path): bool
         }
     }
 
+    public function directoryExists(string $path): bool
+    {
+        return $this->fileExists($path);
```

Comment:
This looks incorrect?

Reply:
According to the Dropbox API it seems correct: https://www.dropbox.com/developers/documentation/http/documentation#files-get_metadata

Also I upgraded to latest version earlier & added a check to see if a directory exists or not before creating it, it seems to be working great.

Reply:
directoryExists should not return true if a file exists at that path, only if a directory exists.